tutorials/applications/QAA to solve a MWIS problem.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<p>I would maybe move this disclaimer further up, maybe right after the first cell with "Let's attempt a simple minimization procedure..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/821/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>